packages/doctor-sdk/tsconfig.json

```diff
@@ -1,6 +1,11 @@
 {
   "extends": "@rsbuild/tsconfig/base",
   "compilerOptions": {
+    "experimentalDecorators": true,
+    "moduleResolution": "node",
+    "removeComments": true,
+    "strict": true,
+    "declaration": true,
```

Comment:
declaration is not used, Modern.js Module will set `declaration` automatically.

Reply:
ok